released/packages/coq-approx-models/coq-approx-models.1.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "damien.pous@ens-lyon.fr"
+version: "1.0"
+
+homepage: "https://gitlab.inria.fr/amahboub/approx-models"
+dev-repo: "git+https://gitlab.inria.fr/amahboub/approx-models.git"
+bug-reports: "https://gitlab.inria.fr/amahboub/approx-models/issues"
+license: "CECILL-B"
+
+synopsis: "Rigorous approximations with a posteriori verified operations"
+description: """
+This is a Coq library to verify rigorous approximations of univariate
+functions on real numbers. Based on interval arithmetic, this library
+also implements a technique of validation a posteriori based on the
+Banach fixed-point theorem. We moreover provide an implementation of
+verified Chebyshev approximations."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.13" & < "8.14~") | (= "dev")}
```

Comment:
There is no OCaml code in the package, as far as I can tell, so you can hope that Coq is stable enough to avoid having to put an upper bound: `"coq" {>= "8.13"}`. Similarly, I don't think enforcing a very specific version of Coquelicot is needed, so you could just have `"coq-coquelicot" {>= "3.2.0"}`.

Reply:
ok for coq, for coquelicot we are no longer compatible with 3.1.0, where normed spaces were not pointed..

Reply:
Sure, that is why proposed to keep the lower bound, but to not put an upper bound. It would be unfortunate that your library fails to compile with a new version of Coq just because it forbids Coquelicot 3.2.1, for example.

Reply:
ok, concerning Coq, should I keep the | = "dev" ?

Reply:
As far as I know, it is not needed if you have no upper bound. (And if you have an upper bound, it does not make much sense either, since `dev` presumably points to a version higher than allowed by the upper bound. For example, `dev` is already past Coq 8.14.)

Reply:
ok, I've removed it then, and it seems all checks have passed.
thanks a lot @silene for your help,
damien